src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -160,13 +161,21 @@ public override void VisitProperty(IPropertySymbol symbol)
                 AddPunctuation(SyntaxKind.OpenBraceToken);
 
                 AddAccessor(symbol, symbol.GetMethod, SyntaxKind.GetKeyword);
-                AddAccessor(symbol, symbol.SetMethod, SyntaxKind.SetKeyword);
+                // PROTOTYPE(init-only): adjust SymbolDisplayVisitor once we have a public IsInitOnly API
```

Comment:
>// PROTOTYPE(init-only): adjust SymbolDisplayVisitor once we have a public IsInitOnly API [](start = 16, length = 89)

Consider moving the comment into the ```IsInitOnly``` helper below, this is where the change should happen.